src/map/skill.cpp

```diff
@@ -7643,6 +7651,18 @@ int skill_castend_nodamage_id (struct block_list *src, struct block_list *bl, ui
 			sc_start(src,bl,type,100,skill_lv,skill_get_time(skill_id,skill_lv)));
 		break;
 
+	//SC_DARKCROW prevents using reflecting skills
+	case CR_REFLECTSHIELD:
+	case MS_REFLECTSHIELD:
+		if (tsc && tsc->data[SC_DARKCROW]) {
+			if (sd)
+				clif_skill_fail(sd, skill_id, USESKILL_FAIL, 0);
+		} else {
+			clif_skill_nodamage(src, bl, skill_id, skill_lv,
+				sc_start(src, bl, type, 100, skill_lv, skill_get_time(skill_id, skill_lv)));
+		}
+		break;
+
```

Comment:
```suggestion
```

Reply:
I'd move it up instead, right before PR_SLOWPOISON and condense it like this so it can fall through

```
	case CR_REFLECTSHIELD:
	case MS_REFLECTSHIELD:
		if (tsc && tsc->data[SC_DARKCROW]) { // SC_DARKCROW prevents using reflecting skills
			if (sd)
				clif_skill_fail(sd, skill_id, USESKILL_FAIL, 0);
			break;
		}
```

Reply:
Thank you. I will change soon

Reply:
@Daegaladh Sorry too late. I just forgot this after I come back home. I applied it.